workbox-config.js

```diff
@@ -1,15 +1,23 @@
+// <<<<<<< HEAD
```

Comment:
can we remove this merge conflict commented or can we remove this commented code by taking a backup of this file

Reply:
+1